draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -2276,7 +2276,16 @@ security properties required by applications.
 strict mode, and when supported by the client. It is, however, prudent to ensure
 that this designation is not the extent of a site's defense against CSRF, as
 same-site navigations and submissions can certainly be executed in conjunction
-with other attack vectors such as cross-site scripting.
+with other attack vectors such as cross-site scripting or abuse of page
+redirections.
```

Comment:
Is the redirection part accurate? My recollection is that we treated A->B->A navigation as cross-site as a mitigation for exactly this risk.

Reply:
Obsoleted by #2750

The initial issue also [links to an attack](https://portswigger.net/web-security/csrf/bypassing-samesite-restrictions#bypassing-samesite-restrictions-using-on-site-gadgets) that makes use of this.